src/Files.App/ServicesImplementation/Settings/UserSettingsService.cs

```diff
@@ -101,7 +101,7 @@ public override bool ImportSettings(object import)
 			return false;
 		}
 
-		private TSettingsService GetSettingsService<TSettingsService>(ref TSettingsService settingsServiceMember)
+		private static TSettingsService GetSettingsService<TSettingsService>(ref TSettingsService? settingsServiceMember)
```

Comment:
Why making it static here? It is relevant to the current PR?

Reply:
I suppose it isn't